### PR TITLE
Update README for about creating issues

### PR DESCRIPTION
**Pull request by ananya7**
ISSUEs are supposed to be created under the main project https://github.com/openebs/openebs

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ We welcome your feedback and contributions in any form possible.
   - Already signed up? Head to our discussions at [#openebs-users](https://openebs-community.slack.com/messages/openebs-users/)
 - Want to raise an issue?
   - If it is a generic (or `not really sure`), you can still raise it at [issues](https://github.com/openebs/openebs/issues)
-  - Project specific issues can be raised at individual project level.
+  - Project specific issues can be raised at github.com/openebs/openebs/issues with the individual project tag.
```
Comment (mahebbar) [inline README.md]: [issues] (https://github.com/openebs/openebs/issues) with the individual project tag.

File: README.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ We welcome your feedback and contributions in any form possible.
   - Already signed up? Head to our discussions at [#openebs-users](https://openebs-community.slack.com/messages/openebs-users/)
 - Want to raise an issue?
   - If it is a generic (or `not really sure`), you can still raise it at [issues](https://github.com/openebs/openebs/issues)
-  - Project specific issues can be raised at individual project level.
+  - Project (repo) specific issues also can be raised at [issues](https://github.com/openebs/openebs/issues) and tagged with the individual repo label like *repo/maya*.
 - Want to help with fixes and features?
```
Comment (prateekpandey14) [inline README.md]: Add link of available github label's what we have [here](https://github.com/openebs/openebs/labels)  ?

Comment (kmova) [reply README.md]: done

Comment (prateekpandey14) [reply README.md]: change -->  repo [labels](https://github.com/openebs/openebs/labels) like *repo/maya*

File: README.md [hunk 3]
```diff
@@ -54,9 +54,9 @@ We welcome your feedback and contributions in any form possible.
...
-  - Have a look at [open issues](https://github.com/issues?q=user%3Aopenebs+is%3Aopen)
+  - Have a look at [open issues](https://github.com/openebs/openebs/labels)
```
Comment (prateekpandey14) [inline README.md]: url "https://github.com/openebs/openebs/issues" ?

Comment (kmova) [reply README.md]: showing the labels may be better.. to have the contributor easily pick the issue they want to work on..

Comment (prateekpandey14) [reply README.md]: cool !!!

File: README.md [hunk 4]
```diff
@@ -54,9 +54,9 @@ We welcome your feedback and contributions in any form possible.
   - Already signed up? Head to our discussions at [#openebs-users](https://openebs-community.slack.com/messages/openebs-users/)
 - Want to raise an issue?
   - If it is a generic (or `not really sure`), you can still raise it at [issues](https://github.com/openebs/openebs/issues)
-  - Project specific issues can be raised at individual project level.
+  - Project (repo) specific issues also can be raised at [issues](https://github.com/openebs/openebs/issues) and tagged with the individual repo labels like *repo/maya*.
```
Comment (mahebbar) [inline README.md]: Project (repository)
repo labels = repository labels

File: README.md [hunk 5]
```diff
@@ -54,9 +54,9 @@ We welcome your feedback and contributions in any form possible.
...
 - Want to help with fixes and features?
-  - Have a look at [open issues](https://github.com/issues?q=user%3Aopenebs+is%3Aopen)
+  - Have a look at [open issues](https://github.com/openebs/openebs/labels)
```
Comment (mahebbar) [inline README.md]: Have a look at = See

File: README.md [hunk 6]
```diff
@@ -54,9 +54,9 @@ We welcome your feedback and contributions in any form possible.
...
   - Have a look at [contributing guide](./CONTRIBUTING.md)
```
Comment (mahebbar) [inline README.md]: Have a look at = See

**Pull request merged**